doc/getting-started.md

```diff
@@ -0,0 +1,390 @@
+
+> **Warning:** To query transaction not related to your own wallet, like this example, you will need to enable `-txindex` option.
+
+To get details of the 69th block :
+
+```bash
+# Find block hash from his height
+dogecoin-cli getblockhash 69
+3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b
+
+# Get block data
+dogecoin-cli getblock 3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b
+{
+  "hash": "3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b",
+  "confirmations": 7816,
+  "strippedsize": 190,
+  "size": 190,
+  "weight": 760,
+  "height": 69,
+  "version": 1,
+  "versionHex": "00000001",
+  "merkleroot": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
+  "tx": [
+    "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db"
+  ],
+  "time": 1386475225,
+  "mediantime": 1386475209,
+  "nonce": 3923708672,
+  "bits": "1e0ffff0",
+  "difficulty": 0.000244140625,
+  "chainwork": "0000000000000000000000000000000000000000000000000000000004600460",
+  "previousblockhash": "ffa69e04f928b84f19d84da25fb544340e54dca6c03c33930da245719e61c5ea",
+  "nextblockhash": "44bf8abbbb96d4dcfb95df563e606c37987133ea3e013b23bbddde8d7f905fdd"
+}
+```
+
+See the `tx` field to find the list of all block transactions. Only one exist in the block 69th, a coinbase transaction.
+
+To get transaction details :
+
+```bash
+# Syntax
+dogecoin-cli getrawtransaction txid verbose
+
+# Example
+dogecoin-cli getrawtransaction 695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db 1
+{
+  "hex": "01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0e04d9eea3520101062f503253482fffffffff0100ac6156be23000023210340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624ac00000000",
+  "txid": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
+  "hash": "695ce4208fa7a87ef9e99805b0910dc129058ecdceb5cef7e25f71dcdc7936db",
+  "size": 109,
+  "vsize": 109,
+  "version": 1,
+  "locktime": 0,
+  "vin": [
+    {
+      "coinbase": "04d9eea3520101062f503253482f",
+      "sequence": 4294967295
+    }
+  ],
+  "vout": [
+    {
+      "value": 393004.00000000,
+      "n": 0,
+      "scriptPubKey": {
+        "asm": "0340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624 OP_CHECKSIG",
+        "hex": "210340a42a5ad6c4c0cd5ae539657032e0a359bd3e0f95771f34d71691b13460a624ac",
+        "reqSigs": 1,
+        "type": "pubkey",
+        "addresses": [
+          "D5TjotnkoXekNQBCn54CBWxjEzCJMMe7WS"
+        ]
+      }
+    }
+  ],
+  "blockhash": "3d2def20cd0d3aca148741ef469bda11647a3040d7669c82745d03c728706a8b",
+  "confirmations": 3964556,
+  "time": 1386475225,
+  "blocktime": 1386475225
+}
+```
+
+`vin` and `vout` will give you information about transaction inputs and outputs.
```

Comment:
This is confusing because a coinbase transaction does not contain a valid input:

```suggestion
The `vout` structure will give you information about where the transaction output.
```